public/css/owner.css

```diff
@@ -0,0 +1,50 @@
+.logo-owner{
+    width: 100%;
+    height: 20%;
+    background-color: white;
```

Comment:
dobrze byloby jakbyscie wyciagneli wszystkie kolory do osobnego pliku css w postacji zmiennych
pozniej w kazdym widoku zaimportujecie te zmienne i bedziecie uzywac. pozniej jesli bysmy chcieli zmienic kolorystyke kazdego widoku to zrobimy to w jednym miejscu
https://developer.mozilla.org/pl/docs/Web/CSS/Using_CSS_custom_properties

Reply:
Hmm, ciekawe, nigdy tego nie robiłem..., jak bd czas to to zrobimy, ale najpierw chcieliśmy podstawowo zrobić